airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/playground/commands.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import click
+from pipelines.cli.click_decorators import LazyPassDecorator, click_ignore_unused_kwargs, click_merge_args_into_context_obj
+from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext
+
+pass_pipeline_context: LazyPassDecorator = LazyPassDecorator(ClickPipelineContext)
```

Comment:
@alafanechere @erohmensing This is the new kid on the block for instantiating a pipeline/dagger client